peen/test/routes/tokens.js

```
@@ -0,0 +1,192 @@
+                .set('content-type', 'application/json')
+                .send({token: "blah"})
+                .end((err, res) => {
+                    res.should.have.status(500);
+                    done()
+                })
+        });
+
+        it('it should return an error if user_id AND token are not provided', (done) => {
+            chai.request(server)
+                .post('/v1/tokens/')
+                .set('content-type', 'application/json')
+                .end((err, res) => {
+                    res.should.have.status(500);
+                    done()
+                })
+        })
+
+        it('it should return an error if token is not a string', (done) => {
+            chai.request(server)
+                .post('/v1/tokens/')
+                .set('content-type', 'application/json')
+                .send({token: {soe: "object"}, user_id: "123"})
+                .end((err, res) => {
+                    res.should.have.status(500);
+                    done()
+                })
+        });
+
+        it('it should return an error if user_id is a string', (done) => {
+            chai.request(server)
+                .post('/v1/tokens/')
+                .set('content-type', 'application/json')
+                .send({token: "string", user_id: "blah"})
+                .end((err, res) => {
+                    res.should.have.status(500);
+                    done()
+                })
+        });
+
+        it('it should return an error if user_id is a float', (done) => {
+            chai.request(server)
+                .post('/v1/tokens/')
+                .set('content-type', 'application/json')
+                .send({token: "string", user_id: "23.24"})
+                .end((err, res) => {
+                    res.should.have.status(500);
+                    done()
+                })
+        });
+
+        it('it should return an error if user_id is a combination of numbers and characters', (done) => {
```

Is this different than 'it should return an error if user_id is a string'?

---

@McGarveyMobile nah you're right and I was just thinking the same thing. This should be covered in the [utils test](https://github.com/getlarky/nudge-peen/pull/1/files#diff-c9dbc4507de87923ac290c3275127033R9).